Basic FE project template/ui-src/styles/components/renting/_renting.scss

```diff
@@ -0,0 +1,120 @@
+.renting {
+  &__container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    @media screen and (max-width: 1160px) {
+      display: flex;
+      flex-wrap: wrap;
+      justify-content: flex-start;
+      align-items: center;
+    }
+
+    @media screen and (max-width: 920px) {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      margin: auto;
+    }
+
+    h1 {
+      @media screen and (max-width: 530px) {
+        font-size: 36px;
+      }
+
+      @media screen and (max-width: 320px) {
+        font-size: 28px;
+      }
+    }
+
+    h5 {
+      @media screen and (max-width: 920px) {
+        font-size: 15px;
+      }
+    }
+  }
+
+  &__small-container {
+    max-width: 440px;
+    width: 100%;
+    margin-right: 15px;
+
+    @media screen and (max-width: 920px) {
+      text-align: -webkit-center;
+      margin: 0;
+    }
+
+    @media screen and (max-width: 500px) {
+      margin: 20px;
+    }
+  }
+
+  &__title {
+    margin-bottom: 45px;
+  }
+
+  &__description {
+    margin-bottom: 40px;
+  }
+}
+
+.btn {
```

Comment:
this is not correct, we have to refactor the html + css here. We talked about BEM, we have here a modifier, you should have used the classes btn btn--text on the button tag.